libraries/AP_InertialSensor/AP_InertialSensor_BMI160.cpp

```diff
@@ -18,8 +18,6 @@
 
 #include <AP_HAL/AP_HAL.h>
 
-#if CONFIG_HAL_BOARD == HAL_BOARD_LINUX
```

Comment:
We probably still want to limit which boards this is built on somehow - e.g. not on our 1MB boards by default.

Wrapping this in a `#if HAL_BMI160_ENABLED` and setting that based on `BOARD_FLASH_SIZE > 1024` in the header (`ifndef`) would be a reasonable starting point there.

Reply:
we shouldn't need to do this as IMU drivers are only included on boards that add the specific driver in their hwdef.dat